modules/layout/react/lib/Layout.tsx

```diff
@@ -53,9 +53,14 @@ const LayoutContainer = styled('div')<LayoutProps>(
   }
 );
 
+export function stripUnit(value: string): number {
+  return parseInt(`${value}`.replace('px', ''), 10);
+}
+
 export default class Layout extends React.Component<LayoutProps> {
   static defaultProps = {
     gutter: canvas.spacing.xs,
+    spacing: stripUnit(canvas.spacing.xs),
```

Comment:
I think you can use `spacingNumbers.xs` from `core` here instead of recalculating it.